### ISSUE TITLE
EIPIP Meeting 103

### ISSUE THREAD

**Issue by poojaranjan** [ethcatherders/EIPIP#334]
# Date and Time

Apr 24, 2024 at 17:30 UTC 

# Location

Zoom: TBA in the Discord #eip-editing channel

YouTube Recording: [EIPIP Meetings](https://www.youtube.com/playlist?list=PL4cwHXAawZxpLrRIkDlBjDUUrGgF91pQw)

# Agenda

### Edit to `Final` Proposals

- [x] https://github.com/ethereum/EIPs/pull/8390

#### Fix Typo
- [x] https://github.com/ethereum/EIPs/pull/8394
- [x] https://github.com/ethereum/EIPs/pull/8156
- [x] https://github.com/ethereum/EIPs/pull/8119
- [ ] https://github.com/ethereum/EIPs/pull/7956

### Update `Stagnant` EIP
- [x] https://github.com/ethereum/EIPs/pull/8418

#### Potential Spam?
- [x] https://github.com/ethereum/EIPs/issues/8492
- [x] https://github.com/ethereum/EIPs/issues/8490
- [x] https://github.com/ethereum/EIPs/issues/8489
- [x] https://github.com/ethereum/EIPs/issues/8482
- [x] https://github.com/ethereum/EIPs/issues/8479
- [x] https://github.com/ethereum/EIPs/issues/8444

### Community proposals added to Issues

- [x] https://ethereum-magicians.org/t/increasing-contract-size-limit-with-increasing-gas-cost-beyond-24-5kb/19708/5  Ref: [EthCatHerders Discord](https://discord.com/channels/916516304972824576/916725260664057916/1232079796649988156)

### Call for Input

Call For Input | Status | Inclination/Result | Deadline
-- | -- | -- | --
https://github.com/ethcatherders/EIPIP/issues/327 | Open | @SamWilsn is in favor, requires input from other editors | April 12th, 2024
https://github.com/ethcatherders/EIPIP/issues/329 | Open | requires input from other editors | April 29th, 2024
https://github.com/ethcatherders/EIPIP/issues/331 | Open | @SamWilsn & @lightclient are in favor of merging this pull request. requires input from other editors | April 29th, 2024
https://github.com/ethcatherders/EIPIP/issues/332 | Open | @g11tech is against entertaining such PRs and closing them off, requires input from other editors | May 8, 2024
https://github.com/ethcatherders/EIPIP/issues/333 | Open |  | May 10, 2024

## 2. Other discussions and updates from past meetings

- How to 
  - [x] create a more inclusive environment to increase community participation for upcoming proposals?
  - [x] a placeholder for collaboration/discussion?
  - [x] showcase EIP Editors work/contribution?
  - [x] lower the number of open Issues & Pull Requests?
- [ ]  PR for review by RIP Editors - https://github.com/ethereum/RIPs/pull/17
- [ ] Web Page Rendering to include EIPs, ERCs, RIPs
- Working Group [Charter](https://hackmd.io/@SamWilsn/rk2CG6qqp/edit)

## 3. EIPs Insight - Monthly EIPs status reporting.

Repo | Issues | PRs | Total
-- | -- | -- | --
EIP |   10 |72  |92
ERC | 4  |87  |91
RIP |  1 | 7 | 8

- [EIPsInsight.com](https://eipsinsight.com/insight/2024/4)
- [Hackmd](https://hackmd.io/@poojaranjan/EthereumImprovementProposalsInsight)

## 4. [EIP Editing Office Hour](https://hackmd.io/@poojaranjan/EIPEditingOfficeHour)
- Agenda [37](https://github.com/ethcatherders/EIPIP/issues/335) Video [36](https://www.youtube.com/watch?v=Xi2eFZl9OEg&list=PL4cwHXAawZxqnDHxOyuwMpyt5s8F8gdmO&index=1&pp=iAQB)
  - 14/27 PRs Reviewed
  - 9/14 PRs are merged (as on date)

## 5. Review action items from earlier meetings
- [Summary meeting 102](https://github.com/ethcatherders/EIPIP/issues/326#issuecomment-2048801413)
Action items from the last meeting

## Next Meeting date & time 

May 08, 2024 at 17:30 UTC

**Comment by poojaranjan** [comment]
# Summary
## 1. Discuss Open Issues/PRs, and other topics
### Edit to Final Proposals
#### [Update EIP-152: Fix reference test ethereum/EIPs#8390](https://github.com/ethereum/EIPs/pull/8390)
- waiting on author's approval. No objection by EIP editors.

### Fix Typo
####  [Update EIP-2982: Fix typos ethereum/EIPs#8394](https://github.com/ethereum/EIPs/pull/8394)
Merged

####  [Update EIP-5749: Fix typo EIP-5749 typo ethereum/EIPs#8156](https://github.com/ethereum/EIPs/pull/8156)
Merged

#### [Update EIP-6963: fix typo ethereum/EIPs#8119](https://github.com/ethereum/EIPs/pull/8119)
Merged

####  [Update EIP-627: fix typo ethereum/EIPs#7956](https://github.com/ethereum/EIPs/pull/7956)

**Comment by poojaranjan** [comment]
closing in favor of #336
